docs/References/troubleshooting.md

```diff
@@ -57,3 +57,53 @@ If you are unable to access your cluster via kubectl from the dashboard, check o
   - *Try killing & restarting the prompt pod*
   - *Check logs from Relay Agent to see if there is any error*
   - *SSL certificate mismatch/error, Ensure that you use the correct SSL certificates as the ingress tends to reject incorrect ones*
+
+---
+
+## Frequently Asked Questions
+
+### 1. What is Paralus?
+
+An open source access manager for Kubernetes clusters, Paralus enables teams to provide secure access to Kubernetes clusters. It lets you create and manage access control policies for people, teams and services across multiple Kubernetes clusters without requiring any modifications to your firewall.
+
+Users can simply import K8 clusters into Paralus projects and define rights on a project to project basis. You can also use Paralus to set up any SSO service using GitHub, Azure AD, Okta, etc. so that users can sign-in onto their clusters with the access rights they were given. 
+
+It also records logs for audit and compliance, so you can see who and when did what on your K8s infrastructure. Paralus can be used with a web GUI, CLI, or API.
+
+### 2. Is Paralus a CNCF Project?
+
+Paralus is currently a candidate in the CNCF Sandbox application process.
+
+### 3. How and where can I try it myself?
+
+You can download and install Paralus for free on [GitHub](https://github.com/paralus/paralus). Alternatively, we’ve setup a one-click installer on [Digital Ocean Marketpace](https://marketplace.digitalocean.com/apps/paralus) to make it even easier. We also have several quickstart guides on the the [blog](https://paralus.io/blog/).
+
+### 4. Is there support or an enterprise license available?
+
+Support for Paralus is available via [community Slack](https://join.slack.com/t/paralus/shared_invite/zt-1a9x6y729-ySmAq~I3tjclEG7nDoXB0A). Enterprise licensing and support is offered with Rafay’s Kubernetes Operations Platform.
+
+### 5. How is this different from Teleport?
+
+Teleport while it also has the ability to provide secure access to Kubernetes clusters is a much more clunky solution as it:
```

Comment:
Teleport does have some additional features which Paralus does not, better to mention that as well